docs/user_manual/getting_verified.md

```diff
@@ -32,23 +32,21 @@ because that server is where most game servers are listed.
 Getting registered for the login server run by Griffon26 is an automated process:
 
 1. First log in to the Hirez servers
+1. Join a game server
+1. Open the Chat Console. (This might not have a keybinding, add one through `Settings > Keybindings > Interface`). Note: This console is different to the Unreal Console (~)
+1. Send a private chat message to `@taserverbot` with only the word `authcode` in it:
 
-2. Send a private chat message to `taserverbot` with only the word authcode in it:
+    `@taserverbot authcode`
 
-    `Say> @taserverbot authcode`
-
-3. Now log in to Griffon26's login server ("the community servers") with the in-game
+1. If you get an error saying the player taserverbot is not online, message Griffon26 in Discord
+1. Now log in to Griffon26's login server ("the community servers") with the in-game
    name you have on the Hirez server and a password of your choosing. Note that your
    in-game name could differ from your account name on Hirez servers. On the community
    servers your in-game name will be your account name.
- 
-4. After logging in go to the "Store" menu and select "Redeem promotion"
-
-5. Fill in the authentication code and press OK
-
-6. If you don't get an error message, registration was successful.
-
-7. Restart Tribes Ascend and log in to Griffon26's server again to remove the "unvrf-" prefix.
+1. After logging in go to the "Store" menu and select "Redeem promotion"
+1. Fill in the authentication code and press OK
+1. If you don't get an error message, registration was successful.
+1. Restart Tribes Ascend and log in to Griffon26's server again to remove the "unvrf-" prefix.
```

Comment:
I like to keep the source markdown as readable as possible by manually fixing the numbering and having empty lines between items. Could you restore this?

Reply:
Through past experience it's more difficult to edit fixed numbers, plus it also looks weird when it reaches 10 or more. Changed it back anyway.